constants/chatbotCommands.py

```diff
@@ -681,40 +683,32 @@ async def getPPMessage(userID: int, just_data: bool = False) -> Any:
     currentMap = current_info["beatmap_id"]
     currentMods = current_info["mods"]
     currentAcc = current_info["accuracy"]
+    currentMode = 0
+    currentMisscount = 0
+
+    beatmap = await beatmap_utils.id_from_api(currentMap, should_save=False)
+    if not beatmap:
+        return "Could not retrieve beatmap information for the given map."
```

Comment:
@tsunyoku is your beatmaps service ready for this kind of "fetch by id" request? I'd love to avoid spreading out these API keys more than we already have -- would be good to limit them to this new beatmaps service

Reply:
i don't remember what our conversation ended up being here but the answer to this is yes